### ISSUE TITLE
Stopped working after recent HA updates

### ISSUE THREAD

**Issue by FS1961** [mampfes/hacs_waste_collection_schedule#1370]
Hello, this had been working great for a few months. Now I'm getting "Unknown" in my Lovelace card.

<img width="419" alt="Screenshot 2023-10-30 at 12 28 08 PM" src="https://github.com/mampfes/hacs_waste_collection_schedule/assets/98458806/9b1b5458-7b17-4f19-9287-b99c2a5700c0">

waste_collection_schedule:
  sources:
    - name: landkreis_kusel_de
      args:
        ortsgemeinde: Erdesbach
      calendar_title: "Trash Pickup"
      customize:
        - type: "Glasabfuhr"
          alias: "Glass"
          icon: mdi:sack-outline
        - type: "Bioabfall"
          alias: "Bio"
        - type: "Restmüll"
          alias: "Trash"
        - type: "Wertstoffsäcke"
          alias: "Recycling"
          icon: mdi:recycle
        - type: "Umweltmobil"
          alias: "Recycling van"
          icon: mdi:truck

- platform: waste_collection_schedule
    name: Trash
    details_format: upcoming
    count: 1
    leadtime: 2
    value_template: '{{value.types|join(", ")}} {% if value.daysTo == 0 %}Today{% elif value.daysTo == 1 %}Tomorrow{% else %}in {{value.daysTo}} Days{% endif %}'
<img width="1682" alt="Screenshot 2023-10-30 at 12 26 48 PM" src="https://github.com/mampfes/hacs_waste_collection_schedule/assets/98458806/45cf426f-edd4-4142-9247-5904050e9df4">

Home Assistant 2023.10.5
Supervisor 2023.10.1
Operating System 11.1

Waste Collection version: 1.43.0

**Comment by codebude** [comment]
Same for me, but with abfall_io source. 
The calendar is filled properly, but the sensor are showing state unknown. 

HA version 2023.10.5 with waste collection 1.43.0

**Comment by 5ila5** [comment]
landkreis_kusel_de actually did not return any data anymore I fixed this in #1371 (it's unlikely that this has anything to do with a homeassistant upgrade as this source file was never touched after creation) 
This fix will be part of release 1.44.0 or now as master (git) version (change version in HACS with 3-dot menu -> redownload -> change version to master)


For abfallnavi_de I cannot reproduce this issue for the example configruation there are no events named Papiertonne but when using an alias the sonsor works fine for me. Is your calendar populated? Are there any logs indecating an error?
Woks for me:
```yaml
waste_collection_schedule:
  sources:
    - name: abfallnavi_de
      args:
        service: coe
        ort: Coesfeld
        strasse: Ahornweg
      customize:
        - type: Papier
          alias: Papiertonne
sensor:
  - platform: waste_collection_schedule
    name: Papiertonne
    details_format: "upcoming"
    value_template: "{% if value.daysTo == 0 %}heute{% elif value.daysTo == 1 %}morgen{% else %}in {{value.daysTo}} Tagen{% endif %}"
    types:
      - Papiertonne
```

for abfall_io the same please take a look if your calendar is populated and if there are any logs because the following code seems to work just fine:
```yaml
waste_collection_schedule:
  sources:
    - name: abfall_io
      args:
        key: "8215c62763967916979e0e8566b6172e"
        f_id_kommune: 2999
        f_id_strasse: 1087
sensor:
  - platform: waste_collection_schedule
    name: wasteCollection
    details_format: "upcoming"
    value_template: "{{value.daysTo}}"
```

**Comment by Adminius** [comment]
Hm... as you see, it was'n there, but now It works again... strange.
![image](https://github.com/mampfes/hacs_waste_collection_schedule/assets/17052262/66c00774-2b7d-4050-a823-3afb132dd860)

solved for me. thx!

P.S. calender is now populated
